www/src/pages/docs/50_StoreMapping.md

```diff
@@ -378,53 +369,53 @@ object Formats {
+| `Store<List<T>>.mapByIndex(index: Int): Store<T>`               | Maps a `Store` of a `List<T>` to one element of that list using the index.                                                                                |
+| `Store<Map<K, V>>.mapByKey(key: K): Store<V>`                   | Maps a `Store` of a `Map<T>` to one element of that map using the key.                                                                                    |
+| `Store<T?>.mapNull(default: T): Store<T>`                       | Maps a `Store` of a nullable `T` to a `Store` of a definitely none nullable `T` using a default value in case of `null` in source-store.                  |
+| `MapRouter.mapByKey(key: String): Store<String>`                | Maps a `MapRouter` to a `Store`. See [chapter about routers](/docs/routing/#maprouter) for more information.                                              |
 
 ### Summary Lens-Factories
 
 | Factory                                                                   | Use case                                                                                        |
 |---------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------|
 | `lensOf(id: String, getter: (P) -> T, setter: (P, T) -> P): Lens<P, T>`   | Most generic lens (used by `lenses-annotation-processor`. Fits for complex model destructuring. |
 | `lensOf(parse: (String) -> P, format: (P) -> String): Lens<P, String>`    | Formatting lens: Use for mapping into `String`s.                                                |
-| `lensForElement(element: T, idProvider: IdProvider<T, I>): Lens<List, T>` | Select one element from a list of entities, therefore some stable Id is needed.                 |
-| `lensForElement(index: Int): Lens<List, T>`                               | Select one element from a list by index. Useful for value objects                               |
-| `lensForElement(key: K): Lens<Map<K, V>, V>`                              | Select one element from a map by some key.                                                      |
+| `lensForElement(element: T, idProvider: IdProvider<T, I>): Lens<List, T>` | Select one element from a list of entities, therefore a stable Id is needed.                    |
+| `lensForElement(index: Int): Lens<List, T>`                               | Select one element from a list by index. Useful for value objects.                              |
+| `lensForElement(key: K): Lens<Map<K, V>, V>`                              | Select one element from a map by key.                                                           |
 
 ## Advanced Topics
 
-### Reactive Rendering of Lists of Entities with automatically Mapped Element Store
+### Reactive Rendering of Entity-Lists With Auto-Mapped Store
```

Comment:
The "element" should be kept! This is the clue of this special feature after all.